docs/asciidoc/modules/ROOT/pages/background-operations/triggers.adoc

```diff
@@ -6,32 +6,27 @@
 Triggers allow the registration of Cypher queries that are called when data in Neo4j is changed (created, updated, deleted).
 You can run them before or after commit.
 
+[WARNING]
+====
+The listeners are eventually consistent,
```

Comment:
Normally a user will not have any context of what a listener is. I think it would be better worded as: installing, updating or removing a trigger is an eventually consistent operation.

Reply:
Changed